### ISSUE TITLE
Error when multiple outdrives have the exact same free space

### ISSUE THREAD

**Issue by zpbd** [tydeno/Ploto#47]
I'm not familiar with PowerShell so apologies for not attempting to fix and raising a PR, but I noticed with two newly formatted and fresh output drives of identical capacity, the script was failing to run because it was passing both drive letters as a parameter.

I think the culprit may be that they both have the exact same free space:

```
$max = ($PlottableOutDrives | measure-object -Property FreeSpace -maximum).maximum
$OutDrive = $PlottableOutDrives | Where-Object { $_.FreeSpace -eq $max}
```

Again I may be misinterpreting as I'm not familiar with the language.

**Comment by tydeno** [comment]
@zpbd Could be as it cannot determine .maximum (has potentially 2 values). I do have several disks that are exact the same size and model and never encouneterd that, tbh. Whats the error/output you receive?

**Comment by tydeno** [comment]
@zpbd Might be fixed in 709adb7
Pushed 709adb7 to branch https://github.com/tydeno/Ploto/tree/potential-fix-for-34-and-45 for testing

BTW: Many thanks for taking the time to report the issues!

**Comment by tydeno** [comment]
Is fixed in latest commits on testing , but hard to reproduce/test. For testing: https://github.com/tydeno/Ploto/tree/testing
Will be kept open for a day, when I have confirmation that testing branch is stable.
